firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -473,4 +500,19 @@ private boolean isInstalledRelease(AppDistributionReleaseInternal latestRelease)
             ReleaseIdentificationUtils.extractInternalAppSharingArtifactId(
                 firebaseApp.getApplicationContext()));
   }
+
+  private void redirectToPlayForAabUpdate(String downloadUrl)
+      throws FirebaseAppDistributionException {
+    if (downloadUrl == null) {
+      throw new FirebaseAppDistributionException(
+          "Download URL not found.", FirebaseAppDistributionException.Status.NETWORK_FAILURE);
+    }
+    Intent updateIntent = new Intent(Intent.ACTION_VIEW);
+    Uri uri = Uri.parse(downloadUrl);
+    updateIntent.setData(uri);
+    updateIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+    currentActivity.startActivity(updateIntent);
+    updateAppTaskCompletionSource.setResult(
```

Comment:
We should probably use `AutoValue` for updateState (similar to AppDistributionRelease)